app/src/main/res/values/donottranslate_questionnaire_values.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Understood. I think it's unnecessary because strings.xml is enough.
But it's okay to merge to `questionnaire` branch!